src/main/java/seedu/duke/command/Command.java

```diff
@@ -0,0 +1,9 @@
+package seedu.duke.command;
+
+import seedu.duke.record.RecordList;
+import seedu.duke.storage.Storage;
+import seedu.duke.ui.Ui;
+
+public abstract class Command {
```

Comment:
![image](https://user-images.githubusercontent.com/61925875/113077380-9fe3dd80-9203-11eb-8139-1b18be9d9fd7.png)

Could this be hard to read/understand? Maybe just zoom in on one of the classes as an example e.g. AddCommand, since the purpose is just to describe that all these command classes have an attribute COMMAND_*?

Reply:
Thanks for pointing this out! PlantUML limitations, decided to split the diagram into 2 images.